pkg/io/io.go

```diff
@@ -65,7 +65,8 @@ func fetchFromRemote(src string) ([]byte, error) {
 }
 
 func fetchFromLocal(src string) ([]byte, error) {
-	f, err := ioutil.ReadFile(src)
+	localSrc := filepath.Join(env.Config().GetString("WorkDir"), env.Config().GetString("Source"), src)
```

Comment:
"Source" is now "Hostname":) 
Does this mean that we always ask user for hostname? If path to config files is relative to `workDir`/`Hostname` then we should add default values to survey when asking for crio, etcd, etc. configs.

Reply:
Oh yeah, not source anymore, ;) . Thanks!

Yes we always need to ask for Kubeconfig first and from there the cluster. If the user selects prompt then we need to obtain the equivalent.

Default values to survey are absolute path already, but relative to the hostname base, I don't think we need to do anything about that besides as long as it's clear enough in the Readme file.

Reply:
Does it make sense to ask for master hostname if no ssh used later?

Reply:
Well it does makes sense to ask for the name of the host the files belongs to

Reply:
I get it now, in some cases we may want to store configs for multiple clusters.

Reply:
I agree that there should be some kind of predefined location for cluster files, so the user could run the tool for multiple clusters. `Source` looks good to me.